### ISSUE TITLE
Test Issue

### ISSUE THREAD

**Issue by rob-w-craig** [jprog-inc/NichqClientIdExample#1]
User with Organization Name clicked the button, but something unexpected occurred: a result that did not match the user's expectations.

A code change may be necessary. Reproduced on testingbox.jmpsi.gov running build [100 emoji].

**Comment by rob-w-craig** [comment]
I disagree with this assessment of the problem. I believe this is exactly the result that should be expected and it is, in fact, the user's expectation that is incorrect. Do you agree with my real take on this real issue @RoyCarubba?

**Comment by rob-w-craig** [comment]
Adding another comment and tagging myself (@rob-w-craig) just to see what it looks like on the tagged person's end.

**Comment by rob-w-craig** [comment]
Tagging myself again after updating my notification settings @rob-w-craig

**Comment by RoyCarubba** [comment]
I'm going to need to do some personal testing for this issue to confirm the reality of this bug.

> I disagree with this assessment of the problem. I believe this is exactly the result that should be expected and it is, in fact, the user's expectation that is incorrect. Do you agree with my real take on this real issue @RoyCarubba?
